rllib/core/rl_module/apis/inference_only_api.py

```diff
@@ -0,0 +1,56 @@
+import abc
+from typing import List
+
+
+class InferenceOnlyAPI(abc.ABC):
+    """An API to be implemented by RLModules that have an inference-only mode.
+
+    Only the `get_non_inference_attributes` method needs to get implemented for
+    a RLModule to have the following functionality:
+    - On EnvRunners (or when self.config.inference_only=True), RLlib will remove
+    those parts of the model not required for action computation.
+    - An RLModule on a Learner (where `self.config.inference_only=False`) will return
+    only those weights from `get_state()` that are part of its inference-only version,
+    thus possibly saving network traffic/time.
+    """
+
+    @abc.abstractmethod
+    def get_non_inference_attributes(self) -> List[str]:
+        """Returns a list of names (str) of attributes of inference-only components.
```

Comment:
This is confusing. The method is named `get_NON_inference_attributes` and the docstring tells that this returns attributes of __inference-only__ attributes.

Reply:
Maybe "Returns a list of names (str) of attributes not used in inference mode."

Reply:
fixed and clarified